src/Raven.Server/Documents/ConflictsStorage.cs

```diff
@@ -56,7 +56,7 @@ private enum ConflictsTable
         static ConflictsStorage()
         {
             Slice.From(StorageEnvironment.LabelsContext, "ChangeVector", ByteStringType.Immutable, out ChangeVectorSlice);
-            Slice.From(StorageEnvironment.LabelsContext, "Id", ByteStringType.Immutable, out IdSlice);
+            Slice.From(StorageEnvironment.LabelsContext, "LowerId", ByteStringType.Immutable, out LowerIdSlice);
```

Comment:
Same issue, this is too generic, need to include the ConflictsId or something like that.